packages/smooth_app/lib/pages/user_management/login_page.dart

```diff
@@ -344,33 +344,78 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
 
   Future<void> showInAppReviewIfNecessary(BuildContext context) async {
     final UserPreferences preferences = context.read<UserPreferences>();
-
     if (!preferences.inAppReviewAlreadyAsked) {
       assert(mounted);
+      final bool? enjoyingApp = await showDialog<bool>(
+        context: context,
+        builder: (BuildContext context) {
+          final AppLocalizations appLocalizations =
+              AppLocalizations.of(context);
 
-      if (await showDialog<bool>(
-            context: context,
-            builder: (BuildContext context) {
-              final AppLocalizations appLocalizations =
-                  AppLocalizations.of(context);
+          return SmoothAlertDialog(
+            body: Text(appLocalizations.app_rating_dialog_title_enjoying_app),
+            positiveAction: SmoothActionButton(
+              text: appLocalizations
+                  .app_rating_dialog_title_enjoying_positive_actions,
+              onPressed: () async => Navigator.of(context).pop(true),
+            ),
+            negativeAction: SmoothActionButton(
+              text: appLocalizations.not_really,
+              onPressed: () => Navigator.of(context).pop(false),
+            ),
+          );
+        },
+      );
+      if (enjoyingApp != null && !enjoyingApp) {
+        await showDialog<bool>(
+          context: context,
+          builder: (BuildContext context) {
+            final AppLocalizations appLocalizations =
+                AppLocalizations.of(context);
 
-              return SmoothAlertDialog(
-                title: appLocalizations.app_rating_dialog_title,
-                body: Text(appLocalizations.app_rating_dialog_body),
-                positiveAction: SmoothActionButton(
-                  text: appLocalizations.app_rating_dialog_positive_action,
-                  onPressed: () async => Navigator.of(context).pop(
-                    await ApplicationStore.openAppReview(),
-                  ),
-                ),
-                negativeAction: SmoothActionButton(
-                  text: appLocalizations.app_rating_dialog_negative_action,
-                  onPressed: () => Navigator.of(context).pop(false),
+            return SmoothAlertDialog(
+              body: Text(
+                  appLocalizations.app_rating_dialog_title_not_enjoying_app),
+              positiveAction: SmoothActionButton(
+                text: appLocalizations.okay,
+                onPressed: () {
+                  // TODO(omegaviv): implement feedback form and link here,https://github.com/openfoodfacts/smooth-app/issues/3419
+                  Navigator.of(context).pop(true);
```

Comment:
Until this feedback form is implemented lets ask them to send a email to `contact @ openfoodfacts.org`. That should be quick to implement, we already have a package for this installed.

Otherwise we would ask them to give (negative) feedback but don't say how, so likely they will write a bad review

Reply:
Or always open the english form for now

Reply:
so should I redirect them to email directly at the press of a button?

Reply:
Yeah, sounds good

Reply:
With a small comment, that the button is the way to add feedback